### PR TITLE
feat(Forms): add `label` and `showLabel` prop to Form.SubmitIndicator

### PR DESCRIPTION
**Pull request by tujoworker**
This PR adds two new properties to the `SubmitIndicator`:

- `showLabel` makes to the label gets visible.
- `label` is to customize the label content.

The `SubmitIndicator` is not effected from this change, except that we speed up the animation.

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Form/SubmitIndicator/SubmitIndicatorDocs.ts [hunk 1]
```diff
@@ -6,6 +6,16 @@ export const SubmitIndicatorProperties: PropertiesTableProps = {
     type: 'string',
     status: 'required',
   },
+  showLabel: {
+    doc: 'If `true`, the label will be shown.',
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Form/SubmitIndicator/SubmitIndicatorDocs.ts:10]:
```suggestion
    doc: 'If `true`, the label will be shown. Defaults to `false`',
```

Not sure how important it is to add, but thought it's nice to say that it's by default false.

File: packages/dnb-eufemia/src/extensions/forms/constants/locales/nb-NO.ts [hunk 2]
```diff
@@ -15,6 +15,9 @@ export default {
       text: 'Send',
       sendText: 'Send inn',
     },
+    SubmitIndicator: {
+      label: 'Under arbeid',
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/constants/locales/nb-NO.ts:19]: Now that we've added translations to this component, we could also add the translationstable in properties docs.

Comment (langz) [reply packages/dnb-eufemia/src/extensions/forms/constants/locales/nb-NO.ts:19]: I added a commit for this https://github.com/dnbexperience/eufemia/pull/4409/commits/8ed7164c0c6dda5719bf9478e9050580d369ecb9

**Review by langz — APPROVED** [review]

File: packages/dnb-eufemia/src/extensions/forms/constants/locales/nb-NO.ts (hunk 2 above)
Comment (langz) [reply packages/dnb-eufemia/src/extensions/forms/constants/locales/nb-NO.ts:19]: AFter the rebase, I've added a new commit with this, https://github.com/dnbexperience/eufemia/pull/4409/commits/7f671450bcaf3700f37016ec4a21d93ecfdc5f52

**Review by langz — APPROVED** [review]
Should we add a screenshot test for this feature?
Or perhaps, it's enough, and/or will indirectly be covered at a later stage in a different feature that will use this feature.

File: packages/dnb-eufemia/src/extensions/forms/constants/locales/nb-NO.ts (hunk 2 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/extensions/forms/constants/locales/nb-NO.ts:19]: Added a visual test 👍

**Comment by tujoworker** [comment]
> Should we add a screenshot test for this feature?

Added a visual test 👍

**Pull request merged**